springwolf-asyncapi/src/main/java/io/github/stavshamir/springwolf/asyncapi/v3/model/channel/ChannelObject.java

```diff
@@ -51,7 +52,8 @@ public class ChannelObject extends ExtendableObject implements Channel {
      * this channel MUST be valid against one, and only one, of the message objects defined in this map.
      */
     @JsonProperty(value = "messages")
-    private Map<String, Message> messages;
+    @Builder.Default
+    private Map<String, MessageReference> messages = new HashMap<>();
```

Comment:
NBC: `messages` are not mandatory. I don't see any reason for a channel to **not** have an associated message, but based on the spec, it's not a required field.